src/main/java/org/shanerx/tradeshop/TradeShop.java

```diff
@@ -91,6 +101,44 @@ public void onEnable() {
 		getCommand("tradeshop").setExecutor(new Executor(this));
 	}
 	
+	private void addMaterials()
+	{
+	   ArrayList<Material> AllowedOld = Arrays.asList(Material.CHEST, Material.TRAPPED_CHEST, Material.DROPPER, Material.HOPPER, Material.DISPENSER);
```

Comment:
Furthermore, I have the feeling that there is some weird indentation here. Can you use your IDE's reformat/refactor/whatever feature to fix it?
(Code -> Refactor for IntelliJ IDEA, Source -> Correct Indentation for Eclipse)